diff --git a/dist/components/tokenizer/index.js b/dist/components/tokenizer/index.js
index 23e88a0937ff74ef45b824f65a0138f3b88ba54a..06100c2f05770cc64858f06a0fbfd15cab49ff99 100644
--- a/dist/components/tokenizer/index.js
+++ b/dist/components/tokenizer/index.js
@@ -68,9 +68,9 @@ export async function createTokenizer(config = {}) {
             try {
                 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
                 // @ts-ignore This fails when verifying CJS but it's actually correct
-                const stemmersPath = import.meta.url.endsWith('ts') ? '../../stemmers/lib' : '../stemmers';
-                const stemmerImport = await import(`../${stemmersPath}/${STEMMERS[config.language]}.js`);
-                stemmer = stemmerImport.stemmer;
+                //const stemmersPath = import.meta.url.endsWith('ts') ? '../../stemmers/lib' : '../stemmers';
+                //const stemmerImport = await import(`../${stemmersPath}/${STEMMERS[config.language]}.js`);
+                //stemmer = stemmerImport.stemmer;
             } catch (e) {
                 throw createError('BUNDLED_ORAMA', config.language);
             }